components/rename_channel_modal/rename_channel_modal.jsx

```diff
@@ -189,7 +187,7 @@ export class RenameChannelModal extends React.PureComponent {
 
         this.setState(state);
 
-        if (state.invalid || (oldName === channel.name && oldDisplayName === channel.display_name)) {
```

Comment:
If the names didn't change, can we exit early without even sending a request to the server?

Reply:
Great point.

Reply:
fixed